src/components/project/project.rules.ts

```diff
@@ -458,10 +462,11 @@ export class ProjectRules {
               label: 'Discuss Suspension',
             },
             {
-              to: await this.getMostRecentPreviousStep(id, [
-                ProjectStep.Active,
-                ProjectStep.ActiveChangedPlan,
-              ]),
+              to: await this.getMostRecentPreviousStep(
+                id,
+                [ProjectStep.Active, ProjectStep.ActiveChangedPlan],
+                changeset
+              ),
```

Comment:
Could we do a wrapping function within this scope?
```ts
const mostRecentPreviousStep = (...steps: ProjectStep[]) =>
  this.getMostRecentPreviousSteep(id, steps, changeset);
```

Reply:
Do you mean like below? 🤔 
```
to: this.mostRecentPreviousStep(
     id,
    [ProjectStep.Active, ProjectStep.ActiveChangedPlan],
    changeset
),
```